JMMServer/JMMServiceImplementationKodi.cs

```diff
@@ -639,6 +640,9 @@ private System.IO.Stream GetGroupsFromFilter(int userid, string GroupFilterId)
                                         }
                                         v.OriginalTitle = v.OriginalTitle.Substring(0, v.OriginalTitle.Length - 1);
                                         //proper naming end
+                                        List<AnimeSeries> sers = grp.GetAllSeries();
+                                        AnimeSeries ser = sers[0];
+                                        v.totalLocal = ser.GetAnimeEpisodesCountWithVideoLocal();
```

Comment:
Are you sure? because your increase this, and mine took the value once. Also I did test my code with this and it look ok, if your sum function also work intention way ?